assets/src/apps/authoring/components/RightMenu/RightMenu.tsx

```diff
@@ -36,9 +37,30 @@ const RightMenu: React.FC<any> = (props) => {
   };
 
   const screenPropertyChangeHandler = (properties: any) => {
-    console.log('SCREEN PROP CHANGED', { properties });
+    const modelChanges = transformScreenSchema(properties);
+    console.log(modelChanges);
+    const screenChanges = {
+      ...currentActivity?.content?.custom,
+      ...modelChanges
+    };
+    if(currentActivity){
+      const clone = JSON.parse(JSON.stringify(currentActivity));
```

Comment:
I think just yesterday a common function was added for this
https://github.com/Simon-Initiative/oli-torus/blob/master/assets/src/utils/common.ts#L21
(maybe rename to `const clonedActivity` since the method is called `clone`)